### ISSUE TITLE
Method for Running ESLint with NPM Module Presets

### ISSUE THREAD

**Issue by atomdmac** [neoclide/coc-eslint#76]
When developing Node.js apps, it's pretty common for a project to use NPM packages to define presets for linting settings.  When a project is "docker-ized", the `npm_modules` directory is often installed to a Docker volume rather than within the host system.  This has the drawback of making the presets unavailable to the host machine that Vim is running on.

I saw that ALE offers a `g:ale_command_wrapper` setting which can allow you to write a script that will run the linter within Docker.  An example can be found [here](https://rymc.io/blog/2019/vim-ale-docker-per-project-linting/).  The example is for a Python linter but the same approach works for Node projects when using ALE.

Is there a way to do this with `coc-eslint`?

**Comment by chemzqm** [comment]
No, coc-eslint use eslint module, but ale use eslint from command line.

**Comment by chemzqm** [comment]
You can configure `eslint.nodePath` as folder of your eslint.
